source/administration/command-line-tools.rst

```diff
@@ -659,6 +659,29 @@ mattermost import
   Child Command
```

Comment:
Please include this also up here in the table of contents section.

Reply:
Done, thanks! :)